adding sounds for various new things and fixing some existing issues

valiant8086 opened the pull request:
…Good luck!

valiant8086 committed:
adding 6 buzzsaw hit noises, and a trigger that I sadly cannot test. Good luck!

valiant8086 commented:
partially addresses #13 but don't close it until we figure out what to do with the Vethriss cloaking and potentially someone adds more buzzsaw noises. I have sounds for bouncing, but I don't have a string for it and am not sure if there even is a string for that, may be when they fire it just never says it hits anybody. It's been a while since I've been in that situation.

stickbear2015 commented:
I wonder if we can ask on newbie or put up something on the boards about it. Since I marked you as a contributor @valiant8086 I have no problem with you reaching out and saying hey, I help out with the officially sanctioned soundpack, here's what we're missing is their text for this this and this? I'm not at home at the moment so I can't just unidle on the game and do it myself.

valiant8086 committed:
adding sounds for shields going up or down on continuance lineage ships.

The sounds are added and the script is updated to trigger them, but untested.

valiant8086 committed:
hopefully fix laser sounds.

One of the strings were rewritten (lasers don't make the ship jerk!) so, if I did this thing right, the trigger will work for the new string.

valiant8086 commented:
if it works, will fix issue #14
